### ISSUE TITLE
TWW 11.0 Prepatch Error

### ISSUE THREAD

**Issue by Bareno** [Athou/ItemLinkLevel#9]
```
9x ItemLinkLevel/main.lua:273: attempt to call global 'InterfaceOptions_AddCategory' (a nil value)
[string "@ItemLinkLevel/main.lua"]:273: in function <ItemLinkLevel/main.lua:173>

Locals:
self = ItemLinkLevel {
}
event = "PLAYER_LOGIN"
panel = OptionsPanel {
 name = "ItemLinkLevel"
}
subtypeCheckBox = subtypeCheckBox {
 Text = subtypeCheckBoxText {
 }
 text = subtypeCheckBoxText {
 }
}
subtypeShortCheckBox = subtypeShortCheckBox {
 Text = subtypeShortCheckBoxText {
 }
 text = subtypeShortCheckBoxText {
 }
}
equipLocCheckbox = equipLocCheckbox {
 Text = equipLocCheckboxText {
 }
 text = equipLocCheckboxText {
 }
}
iLevelCheckbox = iLevelCheckbox {
 Text = iLevelCheckboxText {
 }
 text = iLevelCheckboxText {
 }
}
itemIconCheckbox = itemIconCheckbox {
 Text = itemIconCheckboxText {
 }
 text = itemIconCheckboxText {
 }
}
triggerLootsCheckbox = triggerLootsCheckbox {
 Text = triggerLootsCheckboxText {
 }
 text = triggerLootsCheckboxText {
 }
}
triggerChatCheckbox = triggerChatCheckbox {
 Text = triggerChatCheckboxText {
 }
 text = triggerChatCheckboxText {
 }
}
triggerQualityLabel = triggerQualityLabel {
}
triggerQualityValue = triggerQualityValue {
}
triggerQualitySlider = iLevelTriggerQualityValue {
 NineSlice = Frame {
 }
 Thumb = iLevelTriggerQualityValueThumb {
 }
 Text = iLevelTriggerQualityValueText {
 }
 High = iLevelTriggerQualityValueHigh {
 }
 Low = iLevelTriggerQualityValueLow {
 }
}
(*temporary) = nil
(*temporary) = OptionsPanel {
 name = "ItemLinkLevel"
}
(*temporary) = "attempt to call global 'InterfaceOptions_AddCategory' (a nil value)"
```

Happens on log in

**Comment by Athou** [comment]
I'm sorry I'm not playing anymore :( If you have a patch I will gladly merge a PR.

**Comment by Athou** [comment]
Fixed in 3.0.0!
